btk/measure.py

```diff
@@ -1,175 +1,234 @@
-from abc import ABC
-
+"""Contains examples of functions that can be used to apply a measurement algorithm to the blends
+ simulated by BTK. Every measurement function should take as an input a `batch` returned from a
+ DrawBlendsGenerator object (see its `__next__` method) and an index corresponding to which image
+ in the batch to measure.
+
+It should return a dictionary containing a subset of the following keys/values:
+    - catalog (astropy.table.Table): An astropy table containing measurement information. The
+                                     `len` of the table should be `n_objects`. Currently the
+                                     following column names are supported:
+                                        - dx: horizontal centroid position in pixels.
+                                        - dy: vertical centroid positoin in pixels.
+                                        - ra: object centroid right ascension in arcseconds,
+                                        following the convention from the `wcs` object included in
+                                        the input batch.
+                                        - dec: vertical centroid position in arcseconds,
+                                        following the convention from the `wcs` object included in
+                                        the input batch.
```

Comment:
You should specify that dx and dy should be used when there is a single survey and ra and dec when you have multiresolution.

Reply:
Also, users should feel free to integrate any measurements they want in the catalog field, or even as other keys in the dictionnary ; those keys are mostly here to ensure we have a standardized format in the metrics, but they can have other measurements not covered by the metrics if they want.